### ISSUE TITLE
Missing Accessibility Info

### ISSUE THREAD

**Issue by zcdunn** [mdn/content#37465]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/API/Popover_API

### What information was incorrect, unhelpful, or incomplete?

There was no mention of accessibility semantics or how popovers are exposed to screen readers

### What did you expect to see?

1. How `popovertarget` / `popover` are exposed to screen readers
2. Do developers still need to use `aria-controls`/`aria-haspopup` on the marked button
3. Do developers still need to apply an `aria-role` to the element with the `popup` attribute

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/api/popover_api`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/Popover_API
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/api/popover_api/index.md
* Last commit: https://github.com/mdn/content/commit/e2ac6a94e71034b56a74142619e75e44140918e9
* Document last modified: 2024-11-22T01:13:24.000Z

</details>

**Comment by kantoche** [comment]
Information on accessibility can be found at https://developer.mozilla.org/en-US/docs/Web/API/Popover_API/Using, in the Popover accessibility features section, but it raises several questions: 
- Where does this information come from? No similar information is provided in the HTML specification (https://html.spec.whatwg.org/multipage/popover.html).
- Why is an aria-expanded attribute used when this state is mapped by the open attribute on other components such as summary/details?
- Why is the aria-details attribute used to provide a description accessible to the trigger element, when the content of the controlled element (popover) could be very rich and render this description irrelevant? A popover is not a tooltip
- Why is the tab navigation order updated when the popover is displayed but not the screen reader reading sequence? Its user can't then know when he reaches the popover content. 

Could the MDN team provide any answers?
